include/ck_tile/ops/gemm/pipeline/tile_gemm_shape.hpp

```diff
@@ -8,11 +8,24 @@
 namespace ck_tile {
 
 template <index_t kMPerTile, index_t kNPerTile, index_t kKPerTile>
-struct TileGemmShape
-{
+struct TileGemmShape {
     static constexpr index_t kM = kMPerTile;
     static constexpr index_t kN = kNPerTile;
     static constexpr index_t kK = kKPerTile;
 };
 
+template <typename BlockTile_, typename BlockWarps_, typename WarpTile_>
+struct TileGemmShapeNewGemm
```

Comment:
I initialize a new structure here is because Carlus wants to separate the GEMM new modification with the FMHA shape. The FMHA pipeline call the TileGemmShape and the Gemm Pipeline call the TileGemmShapeNewGemm. I will talk to Carlus today to see what's the soultion. Maybe we could add another structure in the FMHA to decouple that.

Reply:
I talked to Carlus. I will merge the two structures together and accommodate the FMHA to the Tile, Block, Warp 3 level structure.

Reply:
Great, that is the solution I'd expect :)